src/test/pybind/test_rados.py

```diff
@@ -891,6 +891,627 @@ def test_applications(self):
         self.ioctx.application_metadata_remove("app1", "key1")
         eq([("key2", "val2")], self.ioctx.application_metadata_list("app1"))
 
+
+class TestIoctx2(object):
```

Comment:
i don't think we need to duplicate the whole `TestIoctx` just for testing `rados.open_ioctx2`.

Reply:
yea! i do not want to take a whole test at the beginning, but i have no idea which ones should be testing, so...your suggestions is appreciate. thanks.